app/logic/emailHandler.py

```diff
@@ -27,6 +28,8 @@ def __init__(self, raw_form_data, url_domain, sender_object):
         self.program_ids = None
         self.recipients = None
         self.sl_course_id = None
+        self.attachment_path = app.config['email']['attachment_path']
+        self.attachmentFullPath = os.path.join(self.attachment_path, request.files['attachmentObject'].filename)

```

Comment:
We need to find another way without directly referencing the `request` module

Reply:
resolved